lib/shared/bucketing-assembly-script/assembly/types/dvcUser.ts

```diff
@@ -257,18 +237,14 @@ export class DVCPopulatedUser extends JSON.Value implements DVCUserInterface {
         return new DVCPopulatedUser(DVCUser.fromUTF8(userBytes))
     }
 
-    getCombinedCustomData(): JSON.Obj {
-        return this._combinedCustomData
-    }
```

Comment:
Can we keep the custom data logic in this class so that we don't mess up the order of checking customData and customData